scripts/globals/abilities/pets/crimson_howl.lua

```diff
@@ -14,8 +14,13 @@ abilityObject.onAbilityCheck = function(player, target, ability)
 end
 
 abilityObject.onPetAbility = function(target, pet, skill, summoner)
-    local bonusTime = utils.clamp(summoner:getSkillLevel(xi.skill.SUMMONING_MAGIC) - 300, 0, 200)
-    local duration = 60 + bonusTime
+    local duration = 0
+    -- Sep. 9, 2010 Update - Blood Pact: Ward duration changes
+    if xi.settings.main.ENABLE_WOTG == 1 then
+        duration = 60 + utils.clamp(summoner:getSkillLevel(xi.skill.SUMMONING_MAGIC) - 300, 0, 200)
+    else
+        duration = math.min(30 + xi.summon.getSummoningSkillOverCap(pet), 180)
+    end
```

Comment:
WoTG durations were OOE for ASB.

You only need
duration = math.min(30 + xi.summon.getSummoningSkillOverCap(pet), 180)